doc/source/releases.rst

```diff
@@ -32,10 +32,17 @@ The initial version, released 2015-11-30.
 0.2
 ---
 
-This is a planned future release. We plan to include:
+This is a planned future release where most of it is already present within the
+development branch. We plan to include:
 
-* A protocol suite that uses the 2PC protocol in SCAPI.
+* The :ref:`TinyTables<tinytables>` protocol suite as well as a :ref:`Dummy
+  arithmetic<DUMMY_ARITHMETIC>` protocol suite.
 
-* A persistent storage alternative to the current MySQL storage, but
-  that is instead based on files and that can efficiently handles very
-  large amounts of preprocessed data.
+* The preprocessing phase for the SPDZ protocol suite. 
+
+* Many improvements to the standard library. See the :ref:`full list<STD_LIB>`.
+
+* Rework of the entire framework to ease the writing and understanding of
+  applications.
```

Comment:
What is the exact plan here

I would prefer to only make promises we keep?

Reply:
Not sure I understand what you are asking. The rework is already done, and eases writing and readablility of applications, no?

Reply:
The text is slightly confusion : "This is a planned future release"
I thought it indicated more work.

Maybe write: Next release

And then have a section of future work/releases. Maybe that could be empty?